README.md

```
@@ -23,20 +23,14 @@
-        <img alt="Static Badge" src="https://img.shields.io/badge/OpenUPM-v5.0.0-Blue?logo=UPM&color=%2378f1c8">
-    </a>
-    <a href="https://github.com/MindPort-GmbH/VR-Builder/releases" target="_blank">
-        <img alt="Static Badge" src="https://img.shields.io/github/downloads/MindPort-GmbH/VR-Builder/total.svg">
-    </a>
-    <a href="https://github.com/MindPort-GmbH/VR-Builder/issues?q=is%3Aopen" target="_blank">
-        <img alt="Static Badge" src="https://img.shields.io/github/issues/MindPort-GmbH/VR-Builder?style=flat-square&color=%232EA043&label=open issues">
-    </a>
-    <a href="https://discord.com/invite/aUdwRRPgrK" target="_blank">
-        <img src="https://img.shields.io/discord/861482616539578378" alt="Discord conversation">
-    </a>
+    <a href="https://openupm.com/packages/co.mindport.vrbuilder.core/"><img src="https://img.shields.io/npm/v/co.mindport.vrbuilder.core?label=openupm&amp;registry_uri=https://package.openupm.com" /></a>
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)